lib/extract-page-headings/index.test.js

```diff
@@ -4,23 +4,35 @@ const metalsmith = require('metalsmith')
 const plugin = require('./index.js')
 
 describe('extract-page-headings plugin', () => {
-  it('generated heading metadata matches expected', (done) => {
+  let pages
+  beforeAll((done) => {
     metalsmith('lib/extract-page-headings/fixtures')
       .use(plugin())
       .build((err, files) => {
         if (err) {
           return done(err)
         }
-        Object.keys(files).forEach((file) => {
-          const metadataHeadings = files[file].headings
-          const expectedHeadings = [
-            { depth: 1, text: 'Heading level 1', url: 'heading-level-1' },
```

Comment:
For the purpose of avoiding repetition, could you keep this data here and then on the new test on line 29 you could add the aliases by looping over the alias data, eg:

```
const aliases = ['one', 'two', 'three']
aliases.forEach((alias, i) => expectedHeadings[i].aliases = alias)
```

Reply:
The repetition was an intentional decision so if this isn't blocking I think this is read to merge. 👍🏻

Reply:
It's not. Whilst in my opinion it's quite a lot  of data to be repeating, repetition for the sake is readability is at worst not a bad choice.

I'd still like to see the commit history cleaned up though before I approve this.

Reply:
This repository has never had blocking requirements for that you can see current team members with lining fix commits from very recent so let's get this merged.

Reply:
If you think that's important then feel free to do it yourself but I think this is ready to merge.